test/gke/select-cluster.sh

```diff
@@ -31,6 +31,9 @@ echo $cluster > cluster-name
 echo "creating cilium ns"
 kubectl create ns cilium || true
 
+echo "deleting terminating namespaces"
+kubectl get ns | grep Terminating | awk '{print $1}' | xargs -n 1 bash -c 'kubectl get ns -o json $1 | tr -d "\n" | sed "s/\"finalizers\": \[[^]]\+\]/\"finalizers\": []/" | kubectl replace --raw /api/v1/namespaces/$1/finalize -f -' -
```

Comment:
It is probably also worth it to do this in release-cluster since it's a general purpose cleanup in the end

Reply:
It might also be good to explain why it's here and what it fixes (and maybe link to the k8s issue?)

Reply:
good point, done